### PR TITLE
WinUI-2901 : Prepare UG documentation for Rating control in WinUI.

### PR DESCRIPTION
**Pull request by Priyanka-Vijayasankar**
### Description

Prepare UG documentation for Rating control in WinUI.

### CODE REVIEW

File: winui/Rating/Overview.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+title: About WinUI Rating control| Syncfusion
+description: Learn here all about introduction of Syncfusion WinUI Rating Control (SfRating) with key features and more.
+platform: WinUI
+control: SfRating
+documentation: ug
+---
+
+# WinUI Rating Overview
+
+The Syncfusion provides the number of stars that represents a rating. And also used to configure the item size, tooltip and the number of displayed items in the SfRating control. The Rating control can be used in various scenarios like movie rating, rating the application and etc.
```
Comment (Manivannan-E) [inline winui/Rating/Overview.md:12]: @Priyanka-Vijayasankar Change the overview and key features content with the following PR. https://github.com/syncfusion-content/website-whatsnew/pull/367/files

File: winui/Rating/Overview.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+[!Rating control overview in WinUI](Rating_images/winui_rating_overview.png)
+
+## Key features
+
```
Comment (Manivannan-E) [inline winui/Rating/Overview.md:17]: @Priyanka-Vijayasankar Use * symbol to show bulletin for representing key features.

File: winui/Rating/Overview.md (hunk 1 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Overview.md]: Changed the overview and key features content above mentioned PR.

File: winui/Rating/Overview.md (hunk 2 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Overview.md:17]: Used * symbol in the key features.

File: winui/Rating/Overview.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+The .NET WinUI Rating control is used to provide or view ratings on a numeric scale for any service provided, such as movies applications, products, and more.
+
+[!Rating control overview in WinUI](Rating_images/winui_rating_overview.png)
+
+## Key features
+
+* `Precision`: Set an accuracy level for ratings. 
+* `Template`: Add rating items as images, paths, font icons, and other elements.
+* `Read only`: Limit user selection and only allow viewing of the rating value. 
+* `ItemsCount`: Support to determine the number of Rating items to be displayed.
```
Comment (Manivannan-E) [inline winui/Rating/Overview.md:21]: @Priyanka-Vijayasankar To specify the number of items to display.

Comment (Priyanka-Vijayasankar) [reply winui/Rating/Overview.md]: Changed the content as mentioned above.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/Rating/Customization.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Customization with WinUI Rating control | Syncfusion
+description: Learn here all about UI customization with Syncfusion WinUI Rating Control (SfRating), its elements, and more.
```
Comment (OnyimbiNelson) [inline winui/Rating/Customization.md:4]: description: Learn about UI customization with Syncfusion WinUI Rating Control (SfRating), its elements, and more here.

File: winui/Rating/Overview.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: About WinUI Rating control| Syncfusion
+description: Learn here all about introduction of Syncfusion WinUI Rating Control (SfRating) with key features and more.
```
Comment (OnyimbiNelson) [inline winui/Rating/Overview.md:4]: description: Learn all about the introduction of Syncfusion WinUI Rating Control (SfRating) with key features and more here.

File: winui/Rating/Overview.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+platform: WinUI
+control: SfRating
+documentation: ug
+---
+
+# WinUI Rating Overview
+
+The .NET WinUI Rating control is used to provide or view ratings on a numeric scale for any service provided, such as movies applications, products, and more.
```
Comment (OnyimbiNelson) [inline winui/Rating/Overview.md:12]: The .NET WinUI Rating control is used to provide or view ratings on a numeric scale for any service, such as movie applications, products, and more.

File: winui/Rating/Precision.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: Precision with WinUI Rating control | Syncfusion
+description: Learn here all about precision mode defines the accuracy level and it has full, half and exact options of the SfRating control.
```
Comment (OnyimbiNelson) [inline winui/Rating/Precision.md:4]: description: Learn about the precision mode, which defines the accuracy level and has full, half, and exact options of the SfRating control here.

File: winui/Rating/Tooltip.md [hunk 8]
```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: ToolTip with WinUI Rating control | Syncfusion
+description: Learn here all about Enabletooltip and tooltip format support in the rating items in WinUI rating control.
```
Comment (OnyimbiNelson) [inline winui/Rating/Tooltip.md:4]: description: Learn all about Enabletooltip and tooltip format support in the rating items in WinUI rating control here.

File: winui/Rating/Customization.md (hunk 4 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Customization.md:4]: Changed the description as mentioned above.

File: winui/Rating/Overview.md (hunk 5 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Overview.md:4]: Changed the overview content description as mentioned above.

File: winui/Rating/Overview.md (hunk 6 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Overview.md:12]: Changed the content as mentioned above.

File: winui/Rating/Precision.md (hunk 7 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Precision.md:4]: Changed the content description as mentioned above.

File: winui/Rating/Tooltip.md (hunk 8 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Tooltip.md:4]: Changed the content description as mentioned above.

**Pull request merged**